adoc/release-4.5.4.adoc

```diff
@@ -0,0 +1,15 @@
+== Changes in 4.5.4
+
+=== New release of addon images
+
+4.5.4 brings a new release of addon images.
```

Comment:
What has changed in the addon images ?

Reply:
Right, I added that they include SLES patches up to the end of April 2021. Thanks